Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -97,16 +97,16 @@ public final class PerformanceMeter {
     private let metrics: [PerformanceMetric]
     private let delegate: PerformanceMeterDelegate
     private let invocationFile: StaticString
-    private let invocationLine: UInt
+    private let invocationLine: Int
 
-    private init(metrics: [PerformanceMetric], delegate: PerformanceMeterDelegate, file: StaticString, line: UInt) {
+    private init(metrics: [PerformanceMetric], delegate: PerformanceMeterDelegate, file: StaticString, line: Int) {
         self.metrics = metrics
         self.delegate = delegate
         self.invocationFile = file
         self.invocationLine = line
     }
 
-    static func measureMetrics(_ metricNames: [String], delegate: PerformanceMeterDelegate, file: StaticString = #file, line: UInt = #line, for block: (PerformanceMeter) -> Void) {
+    static func measureMetrics(_ metricNames: [String], delegate: PerformanceMeterDelegate, file: StaticString = #file, line: Int = #line, for block: (PerformanceMeter) -> Void) {
```

Comment:
Why not make `metricNames: [PerformanceMetric]` too? As they're strings, we have to map from `PerformanceMetric` to `String` down below.